### PR TITLE
Update closeness centrality

### PR DESCRIPTION
**Pull request by tomasonjo**
I found this cool [publication](http://www.casos.cs.cmu.edu/publications/papers/CMU-ISR-11-113.pdfl), where I stole the docs. How do you like this more technical documentation ? And do you think the "chapters" in closeness centrality, that are filled out need more depth ?

### CODE REVIEW

**Comment by jexp** [comment]
As a general note, can we do one sentence per line, that makes it much easier to read / diff in asciidoc, as changes stay on the same line and are not pushed through the paragraph.

File: doc/betweenness-centrality.adoc [hunk 1]
```diff
@@ -4,13 +4,14 @@ _Betweenness Centrality_ is a measure of centrality in a graph. In graph theory
 
 == History, Explanation
 
-_Betweenness Centrality_ finds wide application in network theory. It represents the degree of which nodes stand between each other. For example, in a telecommunications network, a node with higher betweenness centrality would have more control over the network, because more information will pass through that node. Betweenness centrality was devised as a general measure of centrality. It applies to a wide range of problems in network theory, including problems related to social networks, biology, transport and scientific cooperation.
```
Comment (jexp) [inline doc/betweenness-centrality.adoc]: I really like this first paragraph and wouldn't want to remove it.

File: doc/betweenness-centrality.adoc [hunk 2]
```diff
@@ -4,13 +4,14 @@ _Betweenness Centrality_ is a measure of centrality in a graph. In graph theory
...
-
-Although earlier authors have intuitively described centrality as based on betweenness, Freeman (1977) gave the first formal definition of betweenness centrality. The idea was earlier proposed by mathematician J. Anthonisse, but his work was never published.
-
-
-
-
+_Anthonisse (1971) saw the rush in a graph as the amount a node in a network has to
```
Comment (jexp) [inline doc/betweenness-centrality.adoc]: what is the "rush" ?

File: doc/betweenness-centrality.adoc [hunk 3]
```diff
@@ -4,13 +4,14 @@ _Betweenness Centrality_ is a measure of centrality in a graph. In graph theory
...
+intermediate between other nodes. Freeman (1977, 1979) defined betweenness
+centrality as one of the “three distinct intuitive conceptions of centrality” (Freeman,
+1979: 215). Betweenness centrality is often connected with the notion of control over the
+flow of information. Betweenness centrality is calculated by a breath-first search
+algorithm which calculates the shortest paths from every node to all other nodes (Brandes
+2001). The nodes which lie on these shortest paths are favored in the betweenness
```
Comment (jexp) [inline doc/betweenness-centrality.adoc]: which _most frequently_ lie on these shortest paths

File: doc/closeness-centrality.adoc [hunk 4]
```diff
@@ -1,24 +1,113 @@
 = Closeness Centrality
 
-_Closeness Centrality_ of a node is a measure of centrality in a network, calculated as the sum of the length of the shortest paths between the node and all other nodes in the graph. Thus the more central a node is, the closer it is to all other nodes.
+_Closeness Centrality_ of a node is a measure of centrality in a network, 
+that measures the nearness (as opposite from the distance) from a node to 
+all other nodes. Nodes having a
+high closeness score have short distances to all other nodes.
```
Comment (jexp) [inline doc/closeness-centrality.adoc]: have the shortest distances...

**Comment by tomasonjo** [comment]
updated the PR with requested changes. 
Will format each sentence to its own line in a separate PR, where I will only do the formating and no changes for easier diff reading.

File: doc/betweenness-centrality.adoc [hunk 5]
```diff
@@ -4,13 +4,14 @@ _Betweenness Centrality_ is a measure of centrality in a graph. In graph theory
...
-
+Anthonisse (1971) introduced the concept _rush_ in a graph as the amount a node in a network has to intermediate between other nodes. 
+Freeman (1977, 1979) defined betweenness centrality as one of the “three distinct intuitive conceptions of centrality” (Freeman, 1979: 215). 
+Betweenness centrality is often connected with the notion of control over the flow of information. 
+Betweenness centrality is calculated by a breath-first search algorithm which calculates the shortest paths from every node to all other nodes (Brandes 2001). 
+The nodes which most frequently lie on these shortest paths are favored in the betweenness centrality score. 
 
 
+_Betweenness Centrality_ finds wide application in network theory. It represents the degree of which nodes stand between each other. For example, in a telecommunications network, a node with higher betweenness centrality would have more control over the network, because more information will pass through that node. Betweenness centrality was devised as a general measure of centrality. It applies to a wide range of problems in network theory, including problems related to social networks, biology, transport and scientific cooperation.
```
Comment (jexp) [inline doc/betweenness-centrality.adoc]: newline

File: doc/closeness-centrality.adoc [hunk 6]
```diff
@@ -1,30 +1,114 @@
 = Closeness Centrality
 
-_Closeness Centrality_ of a node is a measure of centrality in a network, calculated as the sum of the length of the shortest paths between the node and all other nodes in the graph. Thus the more central a node is, the closer it is to all other nodes.
+_Closeness Centrality_ of a node is a measure of centrality in a network,
```
Comment (jexp) [inline doc/closeness-centrality.adoc]: no newline after comma :)

File: doc/single-shortest-path.adoc [hunk 7]
```diff
@@ -6,6 +6,17 @@ _All Pairs Shortest Path_ on the other hand calculates a shortest path forest co
 
 == History, Explanation
 
+Path finding, in particular searching in a maze, belongs to the classical graph
```
Comment (jexp) [inline doc/single-shortest-path.adoc]: please one sentence per line.

**Comment by tomasonjo** [comment]
done :)

**Pull request merged**